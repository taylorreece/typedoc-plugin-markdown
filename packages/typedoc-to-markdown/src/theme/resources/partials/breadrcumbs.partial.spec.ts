import { TestApp } from '../../../../test/test-app';

describe(`Breadcrumbs:`, () => {
  let testApp: TestApp;

  describe(`(with readme)`, () => {
    beforeAll(() => {
      testApp = new TestApp();
      testApp.bootstrap();
      console.log(testApp.project);
    });

    test(`should compile README breadcrumbs'`, () => {
      expect('hello').toMatchSnapshot();
    });
  });
});
