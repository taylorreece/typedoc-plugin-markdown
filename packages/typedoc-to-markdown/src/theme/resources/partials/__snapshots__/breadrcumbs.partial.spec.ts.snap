// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Breadcrumbs: (with readme) should compile README breadcrumbs' 1`] = `"hello"`;
